src/components/Hoverable/index.js

```diff
@@ -94,7 +95,7 @@ class Hoverable extends Component {
         if (this.isScrollingRef && this.props.shouldHandleScroll && !this.state.isHovered) return;
 
         if (isHovered !== this.state.isHovered) {
-            this.setState({isHovered}, isHovered ? this.props.onHoverIn : this.props.onHoverOut);
+            this.setState({isHovered}, () => (isHovered ? this.props.onHoverIn : this.props.onHoverOut)(ev));
```

Comment:
Given that `onHoverIn` may be called without events it's better to use `onMouseEnter`. Add onMouseEnter and onMouseLeave props to Hoverable and use onMouseEnter to set the target and mouse position.

In Tooltip:
```js
    const target = useRef(null);
    const initialMousePosition = useRef({x: 0, y: 0});

    const updateTargetAndMousePosition = useCallback((e) => {
        target.current = e.target;
        initialMousePosition.current = {x: e.clientX, y: e.clientY};
    }, []);

    <Hoverable onMouseEnter={updateTargetAndMousePosition) ... />

```